homeassistant/components/daikin/const.py

```diff
@@ -39,6 +44,20 @@
         CONF_DEVICE_CLASS: DEVICE_CLASS_TEMPERATURE,
         CONF_UNIT_OF_MEASUREMENT: TEMP_CELSIUS,
     },
+    ATTR_HUMIDITY: {
+        CONF_NAME: "Humidity",
+        CONF_ICON: "mdi:percent",
+        CONF_TYPE: SENSOR_TYPE_HUMIDITY,
+        CONF_DEVICE_CLASS: DEVICE_CLASS_HUMIDITY,
```

Comment:
I think the `CONF_ICON` should not be set when a `CONF_DEVICE_CLASS` is also set

Reply:
Yes you are right @mlemainque